tests/build/build_templates.go

```diff
@@ -303,8 +305,8 @@ var _ = framework.BuildSuiteDescribe("Build templates E2E test", Label("build",
 				}
 			}
 		})
-
-		It(fmt.Sprintf("triggers PipelineRun for symlink component with source URL %s with component name %s", pythonComponentGitHubURL, symlinkComponentName), Label(buildTemplatesTestLabel, sourceBuildTestLabel), func() {
+		// Temporarily disabled due to https://issues.redhat.com/browse/STONEBLD-2981
```

Comment:
there is some improvement in the CI today (I guess fixes merged in infra-deployment helped improve the situation), not seeing failure for this component now